tests/unit/test_serialization.py

```diff
@@ -310,3 +314,21 @@ async def test_deserialization_fails(default_config_path):
         await deserializer.deserialize(enc_bytes)
 
     assert mock_registry_client.method_calls == [call.get_schema_for_id(1)]
+
+
+@pytest.mark.parametrize(
+    "expected_subject,strategy,subject_type",
+    (
+        (Subject("foo-key"), NameStrategy.topic_name, SUBJECT_VALID_POSTFIX[0]),
```

Comment:
See comment related to this test.

But also this test is more readable if there is no import of `SUBJECT_VALID_POSTFIX` but instead each parametrized test here refers either to `SubjectType.key` or `SubjectType.value`, IMHO.

Reply:
Yeah, you are right. Initially I've wrote the test as you are suggesting, then I thought while adding the `SUBJECT_VALID_POSTFIX` variable, shouldn't we use the same also in test? But it looks terrible, going to revert